src/components/form/fieldCreator.jsx

```diff
@@ -56,7 +56,7 @@ export const createField = ({
             },
             unbind: (model, { value }) => mapping.unbind(model, format.unbind(value)),
             validate: ({ value, isDirty }, ctx) => (
-                (isDirty || ctx.isSubmitted) && (required || isValued(value)) ? fullValidator(value, ctx) : {}
+                (required || isValued(value)) ? fullValidator(value, ctx) : {}
```

Comment:
Then 'isDirty' is not used any more?
Could also use 'Object.assign({}, (required || isValued(value)) && fullValidator(value, ctx)'